collect_app/build.gradle

```diff
@@ -70,8 +70,8 @@ def mapboxToken = secrets.getProperty('MAPBOX_ACCESS_TOKEN', '')
 android {
     compileSdkVersion(28)
 
-    viewBinding {
-        enabled = true
+    buildFeatures {
+        viewBinding true
```

Comment:
I don't think we need this anymore right? It's enabled in the `android` block now.